v1/formats/german-jugend-debattiert.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<debate-format schema-version="2.2">
+    <name xml:lang="en">German Jugend Debattiert</name>
+    <name xml:lang="de">Jugend Debattiert</name>
+    <short-name xml:lang="en">Jugend Debattiert</short-name>
+    <short-name xml:lang="de">Jugend Debattiert</short-name>
+
+    <version>1</version>
+
+    <languages>
+        <language>de</language>
+        <language>en</language>
+    </languages>
+
+    <info xml:lang="de">
+        <region>Deutschland</region>
+        <level>Weiterführende Schulen</level>
+        <used-at>Jugend Debattiert Wettbewerb</used-at>
+        <description>Zwei Zweierteams, Eröffnungsreden (jeweils 2 Minuten) + freie Aussprache (12 Minuten) + Schlussreden (jeweils 1 Minute)</description>
+    </info>
+    <info xml:lang="en">
+        <region>Germany</region>
+        <level>Secondary schools</level>
+        <used-at>Jugend Debattiert Competition</used-at>
+        <description>Two teams of two, opening Speeches (2 minutes each) + free debate (12 minutes) + closing speeches (1 minute each)</description>
```

Comment:
lower case "s" in "Speeches" 😉 
(I'll do it when I merge it if no one else gets to it first, probably tomorrow)

Reply:
my bad, changed it :)